nservicebus/containers/index_content_core_[,5].partial.md

```diff
@@ -5,6 +5,7 @@ Note that the instance of `IBus` is scoped for the lifetime of the container. He
 
 ### Cleanup
 
-When using an external container normally the bus instance is not disposed of manually. If `IBus.Dispose()` is called that would indirectly trigger the container to dispose lifetime scope.
+When using an external container normally the bus instance is not disposed of automatically. The following dispose sequence should be applied if latest container adapter packages are used:
```

Comment:
@danielmarbach - do the changes pertain to Version 6 of NServiceBus? or just different versions of the container packages?